procrastinate/worker.py

```diff
@@ -66,43 +92,50 @@ async def run(self) -> None:
         )
 
         with self.listener(), signals.on_stop(self.stop):
-            await self.single_worker()
+            await asyncio.gather(
+                *(
+                    self.single_worker(worker_id=worker_id)
+                    for worker_id in range(self.concurrency)
+                )
+            )
 
         self.logger.info(
             f"Stopped worker on {self.base_context.queues_display}",
             extra=self.base_context.log_extra(action="stop_worker", queues=self.queues),
         )
         self.notify_event = None
 
-    async def single_worker(self):
+    async def single_worker(self, worker_id: int = 0):
```

Comment:
Do we really need a default value for `worker_id` here?

Reply:
we don't. I think it just simplifies tests, but honnestly, I can get rid of it.

Reply:
Up to you really.